### PR TITLE
https-update

### PR DESCRIPTION
**Pull request by jerry-li-dev**
add steps - need screenshots

### CODE REVIEW

**Comment by jerry-li-dev** [comment]
@shashankawasthi88

Pending screenshots. Still need the steps from here. But, way too much content to copy over. Please specify which specific content to copy over.

https://portswigger.net/burp/documentation/desktop/getting-started/proxy-setup/certificate

**Comment by jerry-li-dev** [comment]
@abhishekc23

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md [hunk 1]
```diff
@@ -95,6 +95,105 @@ The broader development community has published some useful tutorials for settin
...
+See [Capturing requests with Interceptor](/docs/sending-requests/capturing-request-data/interceptor/) for full instructions to set up and use this method.
+
+## Capture HTTPS traffic with Postman's built-in proxy
+
+You can use the Postman's proxy to inspect HTTPS communication from your Android, iOS, Linux, MacOS, and Windows devices. After you enable the proxy feature, follow the instructions below to install the security certificate on the target devices.
+
+### MacOS
+
+1. Go to the following location: **~/Library/Application Support/Postman/proxy**
+1. Double click on **rootCA.crt**
```
Comment (shashankawasthi88) [inline src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:109]: @jerry-li-dev the name of the file is changed. Refer to the docs earlier to find updated file names and screen shots

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md [hunk 2]
```diff
@@ -95,6 +95,105 @@ The broader development community has published some useful tutorials for settin
...
+  [![Select always trust for Postman keychain](https://assets.postman.com/postman-docs/WS-proxy.logs.png)](https://assets.postman.com/postman-docs/WS-proxy.logs.png)
+1. Select **Always Trust** only for **Secure Sockets Layer(SSL)**.
+  [![Select always trust only for SSL option](https://assets.postman.com/postman-docs/WS-proxy.logs.png)](https://assets.postman.com/postman-docs/WS-proxy.logs.png)
+
+After these steps, you should be able to capture HTTPS Requests with the Postman proxy.
+
+### Windows
+
+1. From Windows File Explorer, navigate to **%APPDATA%\Postman\proxy**. Typically, it should be located at: **C:\Users\<user>\AppData\Roaming\Postman\proxy**
+1. Right click on the **rootCA.crt** file and select **Install Certificate**.
```
Comment (shashankawasthi88) [inline src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:122]: Same as above

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md [hunk 3]
```diff
@@ -95,6 +95,105 @@ The broader development community has published some useful tutorials for settin
...
+### CentOS and RHEL distros
+
+1. Copy the **rootCA.crt** certificate file from **~/.config/Postman/proxy** to the **/etc/pki/ca-trust/source/anchors/** directory.
+1. Run the command below in terminal to complete the installation:
+
+    `update-ca-trust extract`
+
+### Ubuntu distros
+
+Please follow the steps in this discussion thread [here](https://askubuntu.com/questions/73287/how-do-i-install-a-root-certificate).
```
Comment (shashankawasthi88) [inline src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:140]: Please copy the steps here as well

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md [hunk 4]
```diff
@@ -31,7 +31,7 @@ warning: false
...
+If you are using APIs to build client-side applications - mobile apps, websites or desktop applications - you might want to see the actual HTTP and HTTPS request traffic that is being sent and received in the application. In some cases, you might discover APIs that are not even documented. Postman gives you tools to see and capture this network traffic easily. You can use the built-in proxy in the Postman native apps or use the [Interceptor extension](/docs/sending-requests/capturing-request-data/interceptor/) for the Postman app.
 
 **Note:** For the Postman native apps, request captures over HTTPS will not work if the website has HSTS enabled. Most websites have this check in place.
```
Comment (jkonrath-postman) [inline src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:36]: Change "HSTS" to "HTTP Strict Transport Security (HSTS)"

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md [hunk 5]
```diff
@@ -95,6 +95,105 @@ The broader development community has published some useful tutorials for settin
...
 
-## Interceptor
+## Use Interceptor to capture requests
 
-See [Capturing requests with Interceptor](/docs/sending-requests/capturing-request-data/interceptor/).
+See [Capturing requests with Interceptor](/docs/sending-requests/capturing-request-data/interceptor/) for full instructions to set up and use this method.
+
+## Capture HTTPS traffic with Postman's built-in proxy
+
+You can use the Postman's proxy to inspect HTTPS communication from your Android, iOS, Linux, MacOS, and Windows devices. After you enable the proxy feature, follow the instructions below to install the security certificate on the target devices.
```
Comment (jkonrath-postman) [inline src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:104]: Should be "macOS" (change this globally)

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md [hunk 6]
```diff
@@ -95,6 +95,105 @@ The broader development community has published some useful tutorials for settin
...
+1. Go to the following location: **~/Library/Application Support/Postman/proxy**
+1. Double click on **rootCA.crt**
+1. Choose **System** from the keychain option and then select **OK**.
+  [![Select System keychains](https://assets.postman.com/postman-docs/WS-proxy.logs.png)](https://assets.postman.com/postman-docs/WS-proxy.logs.png)
+1. Select and open the imported Postman certificate. Then select **Always Trust**. You can change this setting after you have completed the testing for your application.
+  [![Select always trust for Postman keychain](https://assets.postman.com/postman-docs/WS-proxy.logs.png)](https://assets.postman.com/postman-docs/WS-proxy.logs.png)
+1. Select **Always Trust** only for **Secure Sockets Layer(SSL)**.
+  [![Select always trust only for SSL option](https://assets.postman.com/postman-docs/WS-proxy.logs.png)](https://assets.postman.com/postman-docs/WS-proxy.logs.png)
+
+After these steps, you should be able to capture HTTPS Requests with the Postman proxy.
```
Comment (jkonrath-postman) [inline src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:117]: Change "should" to "will" (don't indicate probability - assume our code always works...)

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md (hunk 2 above)
Comment (jkonrath-postman) [inline src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:122]: Change "right click" to "right-click" (check globally)

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md [hunk 7]
```diff
@@ -95,6 +95,105 @@ The broader development community has published some useful tutorials for settin
...
+1. Right click on the **rootCA.crt** file and select **Install Certificate**.
+1. Select the Local Machine and proceed. This will need Administrator permissions.
+1. Select **Place all certificates in the following store**.
+1. Select **Browse** and then select **Trusted Root Certification Authorities**.
+1. Select **OK** and **Next** to confirm the options.
+1. Select **Finish** to save all configurations and import the certificate.
+
+### Linux
+
+### CentOS and RHEL distros
```
Comment (jkonrath-postman) [inline src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:131]: Spell out RHEL to Red Hat Enterprise Linux (we should honor their trademark)

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md [hunk 8]
```diff
@@ -95,6 +95,105 @@ The broader development community has published some useful tutorials for settin
...
+
+    ![download openssl installer](https://assets.postman.com/postman-docs/ios-transfer-cert.jpeg)
+1. Go to **Settings > Profile Downloaded**.
+
+    ![download openssl installer](https://assets.postman.com/postman-docs/ios-install-cert.jpeg)
+1. Select **Install** for both of the following prompts. Installing a certificate requires the user to enter the device passcode to proceed.
+
+    ![download openssl installer](https://assets.postman.com/postman-docs/ios-install-cert2.jpeg)
+
+    When the certificate is installed, your device should show an installation confirmation screen like the one below.
```
Comment (jkonrath-postman) [inline src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:154]: should/will

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md [hunk 9]
```diff
@@ -95,6 +95,105 @@ The broader development community has published some useful tutorials for settin
...
+
+    ![download openssl installer](https://assets.postman.com/postman-docs/ios-root-cert-settings.jpeg)
+
+    ![download openssl installer](https://assets.postman.com/postman-docs/ios-enable-full-trust.jpeg)
+
+## Requirements to use the OpenSSL module
+
+The **OpenSSL** module is internally being used to generate certificate-key pairs. This module has to be installed and accessible through the command line.
+
+Openssl is generally already installed for MacOs and Linux (there can be cases where it is not installed). For windows systems, **OpenSSL** is to be installed if not installed previously.
```
Comment (jkonrath-postman) [inline src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:173]: * Change openssl to OpenSSL
* Change MacOs to macOS

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md [hunk 10]
```diff
@@ -95,6 +95,105 @@ The broader development community has published some useful tutorials for settin
...
+1. From the **System Properties** window, select "Environment Variables."
+
+    ![download openssl installer](https://assets.postman.com/postman-docs/windows-environment-variables-advanced.jpg)
+1. Select **Path** from **User variables** and then select **Edit**.
+
+    ![download openssl installer](https://assets.postman.com/postman-docs/windows-environment-variables-edit-browse-bb.jpg)
+1. Select **Browse**.
+
+    ![download openssl installer](https://assets.postman.com/postman-docs/windows-openssl-edit-environment-variables-bb.jpg)
+1. Go to where the oppssl.exe is. **This PC > Windows (C:) > Program Files > OpenSSL - Win64 > bin**.
```
Comment (jkonrath-postman) [inline src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:193]: Is that the right .exe name?

**Review by jkonrath-postman — CHANGES_REQUESTED** [review]
I left some minor changes - see comments

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md (hunk 10 above)
Comment (abhishekc23) [reply src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:193]: The exe part can be ignored completely as we are just asking the user to select the bin directory

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md (hunk 4 above)
Comment (abhishekc23) [reply src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:36]: This Note can be removed, now that we will be supporting HSTS via HTTPS Proxy

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md (hunk 10 above)
Comment (jerry-li-dev) [reply src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:193]: I have removed this detail from the document.

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md (hunk 4 above)
Comment (jerry-li-dev) [reply src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:36]: Note has been removed.

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md [hunk 11]
```diff
@@ -95,6 +93,137 @@ The broader development community has published some useful tutorials for settin
 * [macOS](https://support.apple.com/en-gb/guide/mac-help/mchlp2591/mac)
 * [Android](https://www.howtogeek.com/295048/how-to-configure-a-proxy-server-on-android/)
 
-## Interceptor
+## Use Interceptor to capture requests
```
Comment (shashankawasthi88) [inline src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:96]: We should move the Interceptor section at the end of this page. Lets complete everything about proxy first and then have Interceptor

**Review by shashankawasthi88 — COMMENTED** [review]
Looks good to me!

**Review by jkonrath-postman — APPROVED** [review]
Looks good to me

File: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md (hunk 11 above)
Comment (sowmyaraj92) [reply src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md:96]: @jerry-li-dev Looks like this comment wasn't addressed. Please ensure we make this change and push it in the next release to prod.

**Pull request merged**